mlflow/store/db_migrations/versions/29599da4828d_add_model_aliases_table.py

```diff
@@ -0,0 +1,42 @@
+"""Add Model Aliases table
+
+Revision ID: 29599da4828d
+Revises: 97727af70f4d
+Create Date: 2023-03-08 13:22:37.174258
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from mlflow.store.model_registry.dbmodels.models import SqlRegisteredModelAlias
+
+# revision identifiers, used by Alembic.
+
+revision = '29599da4828d'
+down_revision = '97727af70f4d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        SqlRegisteredModelAlias.__tablename__,
+        sa.Column(
+            "name",
+            sa.String(length=256),
+            sa.ForeignKey("registered_models.name", onupdate="cascade"),
+            primary_key=True,
+            nullable=False,
+        ),
+        sa.Column("alias", sa.String(length=250), primary_key=True, nullable=False),
+        sa.Column("version", sa.Integer(), primary_key=True, nullable=False),
+        sa.ForeignKeyConstraint(
+            ("name", "version"),
+            ("model_versions.name", "model_versions.version"),
+            onupdate="cascade",
+        ),
+        sa.PrimaryKeyConstraint("name", "alias", name="model_version_alias_pk"),
```

Comment:
I think if you create the PKEY constraint before the ForeignKeyConstraint you might have a better shot of MSSQL Server not complaining :)

Reply:
Oh I was following the ordering for `model_version_tags`:

```
def upgrade():
    op.create_table(
        SqlModelVersionTag.__tablename__,
        sa.Column("key", sa.String(length=250), primary_key=True, nullable=False),
        sa.Column("value", sa.String(length=5000)),
        sa.Column("name", sa.String(length=256), primary_key=True, nullable=False),
        sa.Column("version", sa.Integer(), primary_key=True, nullable=False),
        sa.ForeignKeyConstraint(
            ("name", "version"),
            ("model_versions.name", "model_versions.version"),
            onupdate="cascade",
        ),
        sa.PrimaryKeyConstraint("key", "name", "version", name="model_version_tag_pk"),
    )

```

Reply:
Updated the ordering anyway!